rawspec.c

```diff
@@ -743,33 +801,50 @@ char tmp[16];
             - (ctx.Nts[i]/2) * cb_data[i].fb_hdr.foff
             + (schan % (raw_hdr.obsnchan/raw_hdr.nants)) * // Adjust for schan
                 raw_hdr.obsbw / (raw_hdr.obsnchan/raw_hdr.nants);
-          cb_data[i].fb_hdr.nchans = ctx.Nc * ctx.Nts[i];
-          cb_data[i].fb_hdr.tsamp = raw_hdr.tbin * ctx.Nts[i] * ctx.Nas[i];
+          cb_data[i].fb_hdr.nfpc = ctx.Nts[i];  // Number of fine channels per coarse channel.
+          cb_data[i].fb_hdr.nchans = ctx.Nc * ctx.Nts[i] / raw_hdr.nants; // Number of fine channels.
```

Comment:
Maybe expand the comment, and that of the field in the header/struct definition: `// Number of fine channels, per output (read antenna)`... Probably just me that needs reminding of this being the norm, though

Reply:
We all need reminding, especially with large C/C++ source files.

Reply:
How is this in the fb_hdr_t struct definition for ```nchans```?

```
  // number of fine filterbank channels (as opposed to coarse channels)
  int nchans;
```

The following is already there for the new header field:
```
  // ------------------------ new for FBH5 ------------------------------------
  // Number of fine channels per coarse channel
  // E.g. turbo_seti will compute the number of coarse channels = nchans / nfpc
  int nfpc;

```